### PR TITLE
[bug 1136127] - Users able to post spaced empty message 

### PR DESCRIPTION
**Pull request by mattdeveloper**
* Users are not able to post spaced empty message.

### CODE REVIEW

File: app/js/question_controller.js [hunk 1]
```diff
@@ -166,7 +166,8 @@
     evt.preventDefault();
     var comment = question_field.value;
 
-    if (comment === '') {
+    // Disable submit if the comment is blank or with some spaces
+    if (comment === '' || comment.replace(/\s/g, '') === '') {
```
Comment (Rik) [inline app/js/question_controller.js]: We can do a simpler version of this by using [`trim()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim)

Comment (Rik) [inline app/js/question_controller.js]: You can also ditch the comment because `trim()` is enough to express the intent.

**Comment by mattdeveloper** [comment]
Thanks Rik, I updated it.

**Comment by Rik** [comment]
Landed in https://github.com/mozilla/buddyup/commit/8d28cab3b4eddc409219ae95dac95eb39cebef7a

**Pull request closed without merging**